webapp/src/components/Collapsable/Collapsable.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import debounce from 'lodash.debounce'
+import { t } from '@dapps/modules/translation/utils'
+
+import './Collapsable.css'
+
+export default class Collapsable extends React.PureComponent {
+  static propTypes = {
+    maxHeight: PropTypes.number.isRequired,
+    children: PropTypes.node.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.wrapperElement = null
+    this.debouncedSetCollapsableUsingHeight = debounce(
+      this.setCollapsableUsingHeight,
+      100
+    )
+
+    this.state = {
+      isCollapsed: true,
+      isCollapsable: false
+    }
+  }
+
+  toggleSeeMore = () => {
+    this.setState({ isCollapsed: !this.state.isCollapsed })
+  }
+
+  componentWillMount() {
+    window.addEventListener('resize', this.debouncedSetCollapsableUsingHeight)
```

Comment:
I couldn't see the `see more` until I resize the window. I think that we should call `setCollapsableUsingHeight` once we mount this component and keep the listener just in case the user resize the window.

Also, I think that we need to add a `padding` to the `auction-panel`

<img width="1352" alt="screen shot 2018-11-27 at 15 18 30" src="https://user-images.githubusercontent.com/7549152/49102602-03d2f180-f258-11e8-93ec-ffc979159565.png">

Reply:
Added the padding!
The see more on the land auction wasn't supposed to be there 😅 fixed